examples/tools/structure_graph.py

```diff
@@ -0,0 +1,10 @@
+from bokeh.io import show
+from bokeh.layouts import column
+from bokeh.plotting import figure
+from bokeh.util import structure
+
+# draw the structure graph of a basic figure model
+f = figure(width=400,height=400)
+f.line(x=[1,2,3],y=[1,2,3])
+K = structure.BokehStructureGraph(f)
+show(column(f,K.model))
```

Comment:
I'd suggest making a free function that is the "common" public API, to simplify this. It would just create a `BokehStructureGraph` for you and return the model, as a convenience. Something like:
```
from bokeh.util.structure import generate_structure_plot

sg = generate_structure_plot(f)

show(column(f, sg))
```

If there are not actually any useful configurable things on `BokehStructureGraph` right now then I would even say it could be private `_BokehStructureGraph` for now. That would make the fewer API commitments now, which I think is good (more flexible later)

Reply:
Done